docs/fornax_test/830_release_testplan.md

```diff
@@ -0,0 +1,584 @@
+
+The following tests assume four clusters, denoted as A,B,C and D, are created and configured for tests. A,B,C are kubernetes clusters created using kubeadm, while cluster D is an arktos cluster started by running script arktos-up.sh (https://github.com/CentaurusInfra/arktos/blob/master/hack/arktos-up.sh). These clusters are configured in a hierarchical topology, where Cluster B is an edge cluster to Cluster A, C edge to B, and D edge to C. Detailed instructions on how to configure the hierarchical connection among these clusters are given in the doc test_cluster_setup.md in the same directory as this doc. 
+
+
+## Overview of Test Coverage
+
+The test cases can be classified into three groups. 
+
+**Group I: EdgeCluster Connection**
+
+Test Case 1: Register Hierarchical Edge Clusters with Cloud
+
+Test Case 3: An edge cluster re-connects to the cloud
+
+Test Case 5: An edge cluster continue to work when disconnected from the cloud
+
+Test Case 6: An edge cluster conintues to monintor its sub-edge-clusters when disconnected from cloud
+
+
+
+**Group II: Workload Managment**
+
+Test Case 4: Deploy workload (deployment) to edge clusters using mission
+
+Test Case 7: Deploy workload (job) to edge clusters using mission
+
+Test Case 8: Check mission status using command "kubectl get missions" 
+
+Test Case 9: Update workload to edge clusters using mission
+
+Test Case 10: The mission status of a cluster is "cluster unreachable" if cluster disconnected
+
+Test Case 11: Delete workload to edge clusters using mission
+
+Test Case 12: Deploy workload to an edge cluster with specific name
+
+Test Case 13: Deploy workload to selective edge clusters with given labels
+
+Test Case 15: An edge cluster picks up the new mission added during its disconnection when reconnected to the cloud
+
+Test Case 16: An edge cluster updates the mission updated during its disconnection when reconnected to the cloud
+
+Test Case 17: An edge cluster deletes the mission deleted during its disconnection when reconnected to the cloud
+
+Test Case 18: The mission content deleted in edge cluster will be re-instated
+
+Test Case 19: The mission content changed in edge cluster will be reverted automatically
+
+
+**Group III: Montioring EdgeCluster & Mission States**
+
+Test Case 2: Check edge cluster status when disconnected from the cloud 
+
+Test Case 14: Check edge cluster status when connected from the cloud 
+
+Test Case 20: The status of edge cluster is "Unhealthy" if the clusterd is connected but the underlying cluster is unreachable
+
+
+## Test Cases
+
+**Note: By default, the commands in this doc are run from the root directory of the fornax repo. If not specified, the command is run on the operating machine of Cluster A, which will be referred as "root operator machine" in the rest of this document.**
```

Comment:
"operating machine of Cluster A" -> "master node of cluster A"?

Reply:
I just call it "the machine of Cluster A" now, to be consistent with the other parts of the doc. 

Here I am using one machine for one cluster, so there is not a machine exclusively working as a master node.